docs/README.md

```diff
@@ -401,10 +401,16 @@ messages.
 Yes, you can do it.
 
 1. Ensure the `main` branch in your fork is updated with the latest changes.
-2. Create a GitHub [personal access token][pat] (PAT) and add it as an
-   [encyrpted secret][encyrpted-secret] with name as `BOT_TOKEN`.
-3. Create a branch and make your required changes and send a pull request to
-   your `main` branch.
+2. Ensure there is a `gh-pages` branch in your fork.
+3. You need a publicly accessible OpenShift cluster (See [partner guide to get
+   free access to OCP][partner-ocp]).
+4. Follow the [documentation to create a service account][sa-cluster-token] and
+   corresponding token.  The token should be stored as an [encrypted secret in
+   GitHub repository settings][encrypted-secret] with the key as
+   `CLUSTER_TOKEN`.
```

Comment:
Besides the cluster access token. How would a partner point to her/his specific cluster? How would they set the kube context in the workflow?

Reply:
I just realized `CLUSTER_TOKEN` is not sufficient. We need to make the API server URL also configurable.
I will put this PR on hold for now until that is fixed in the scripts. Thanks for the question!

Reply:
OK, makes sense

Reply:
PR is opened for review: #195